### PR TITLE
Smartcharts 0.4.17

### PR DESCRIPTION
**Pull request by ashkanx**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -924,7 +924,7 @@
     },
     "ansi-colors": {
       "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-1.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-colors/-/ansi-colors-1.1.0.tgz",
```
Comment (negarn) [inline package-lock.json]: why are we changing to http?

Comment (ashkanx) [reply package-lock.json]: no idea. @morteza-binary ?

Comment (negarn) [reply package-lock.json]: should be related to your node version or npm version right? some people will change to http, some people will change to https, which is why we run `npm ci` but still face this when a different person runs `npm i` to install a new package

Comment (morteza-binary) [reply package-lock.json]: @ashkanx I don't have any idea, AFAIK, it was an issue in the older version of NPM but it should be solved.

**Pull request closed without merging**